File: TreasurersApp/ClientApp/src/app/models/ContributionCategory.ts
export class ContributionCategory {
  id: number = null;
  contributionCategoryName: string = null;
  description: string = null;
  displayOrder: number = null;
  active = false;

  constructor(id: number, contributionCategoryName: string, description: string, displayOrder: number, active: boolean) {
    this.id = id;
    this.contributionCategoryName = contributionCategoryName;
    this.description = description;
    this.displayOrder = displayOrder;
    this.active = active;
  }
}

File: TreasurersApp/Models/ContributionCategory.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

namespace TreasurersApp.Models
{
    public partial class ContributionCategory
    {
        public ContributionCategory()
        {
            ContributionType = new HashSet<ContributionType>();
        }

        [JsonProperty("id")]
        public int ContributionCategoryId { get; set; }

        [JsonProperty("contributionCategoryName")]
        public string ContributionCategoryName { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("displayOrder")]
        public int DisplayOrder { get; set; }

        [JsonProperty("active")]
        public bool? Active { get; set; }

        [JsonProperty("contributionTypes")]
        public ICollection<ContributionType> ContributionType { get; set; }
    }
}
